src/fights.ts

```diff
@@ -379,6 +380,24 @@ function embezzlerSetup() {
     }
   });
   if (have($item`License to Chill`) && !get("_licenseToChillUsed")) use($item`License to Chill`);
+  if (
+    globalOptions.ascending &&
+    questStep("questM16Temple") > 0 &&
+    get("lastTempleAdventures") < myAscensions() &&
+    acquire(1, $item`stone wool`, 3 * get("valueOfAdventure") + 100, false) > 0
+  ) {
+    ensureEffect($effect`Stone-Faced`);
+    setChoice(582, 1);
+    setChoice(579, 3);
+    const runSource = findRun();
+    if (runSource) {
+      if (runSource.prepare) runSource.prepare();
+      freeFightOutfit([...(runSource.requirement ? [runSource.requirement] : [])]);
+      while (get("lastTempleAdventures") < myAscensions()) {
```

Comment:
this isn't going to work, you need to get a new runSource every iteration, prepare it, and equip the requirements

Reply:
as an addon to this, I think this is one of the situations like with startDigitize() where you may want to add a backup LTB if you can't find a run